### PR TITLE
Cw/deberta

### PR DESCRIPTION
**Pull request by chris-wells-13**
adding changes to huggingface/models.py, huggingface tests, and setup.py to support deberta.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "tensorflow-estimator==2.7.0",
     "tqdl==0.0.4",
     "psutil==5.7.0",
-    "transformers==4.5.1",
+    "transformers==4.19.2",
```
Comment (benleetownsend) [inline setup.py:31]: this also needs changing in requirements.

Comment (benleetownsend) [reply setup.py:31]: Have tests been run to verify that this change does not break anything else?

**Review by benleetownsend — CHANGES_REQUESTED** [review]
Just need to update dependency in requirements, otherwise looks good to me.

File: setup.py (hunk 1 above)
Comment (chris-wells-13) [reply setup.py:31]: added the change to requirements.txt.  i only ran the huggingface test suite, you think i should run everything (does transformers leak out of there to places i didn't notice)?

Comment (benleetownsend) [reply setup.py:31]: Tokenizers is the main place as we use the HF tokenizers for roberta now.

**Comment by benleetownsend** [comment]
https://github.com/IndicoDataSolutions/finetune/blob/ddedbf10a92475b81ee9f9ea64b755f6b0fde208/tests/test_base_models.py#L569 and https://github.com/IndicoDataSolutions/finetune/blob/development/tests/test_utils.py probably worth running to verify tokenizers.

**Comment by benleetownsend** [comment]
merged as part of #755

**Pull request closed without merging**